package.json

```diff
@@ -19,16 +19,17 @@
   "author": "Grafana Labs",
   "license": "Apache-2.0",
   "dependencies": {
+    "@grafana/async-query-data": "0.0.2",
     "@grafana/experimental": "^0.0.2-canary.39"
   },
   "devDependencies": {
-    "@grafana/aws-sdk": "0.0.37",
-    "@grafana/data": "8.2.1",
-    "@grafana/e2e": "8.2.1",
-    "@grafana/e2e-selectors": "8.2.1",
-    "@grafana/runtime": "8.2.1",
-    "@grafana/toolkit": "8.2.1",
-    "@grafana/ui": "8.2.1",
+    "@grafana/aws-sdk": "0.0.38",
+    "@grafana/data": "8.5.10",
+    "@grafana/e2e": "8.5.10",
+    "@grafana/e2e-selectors": "8.5.10",
+    "@grafana/runtime": "8.5.10",
+    "@grafana/toolkit": "8.5.10",
+    "@grafana/ui": "8.5.10",
```

Comment:
a thought that I have is that this work could maybe (maybe?) be in multiple prs. Like we upgrade all the packages first for example and release that? Nitpicky but just a suggestion if we're feeling nervous at all!